eclipse/rdf4j#2235 first draft of release notes for 3.3.0 (#161)
eclipse/rdf4j-doc: site/content/release-notes/3.3.0.md

---
title: "3.3.0"
---
RDF4J 3.3.0 is a minor release [TODO]

It is backward compatible with previous RDF4J releases.

Highlights:

- [TODO]
- ShaclSail now supports sh:inversePath (limited to single predicates).
-

For a complete overview, see [all issues fixed in 3.3.0](https://github.com/eclipse/rdf4j/milestone/51?closed=1).

## Upgrade notes 

This release is binary compatible with previous releases. 

RDF4J 3.1.x and 3.2.x implicitly included the jackson-core, jackson-databind and jackson-annotations dependencies when using the rdf4j-rio-api maven artifact. In version 3.3.0, these dependencies are explicitly excluded.
Projects relying on this unintended behavior should check if they need to add these jackson dependencies to their build files.  

## Acknowledgements

> The following contributors made this release possible (in alphabetical order): Bart Hanssens, Gaurav Aggarwal, Gauri Bhatanagar, Håvard M. Ottestad, Jeen Broekstra, Navdha Agarwal, Pavel Mihailov, Prince Sachdeva, Raghav Rathi, Reeshabh Kumar Ranjan, Sakshi Saini, Shubhi Jain [TODO] 

Author: ```suggestion
This release was made possible by contributions from Bart Hanssens, Gaurav Aggarwal, Gauri Bhatanagar, Håvard M. Ottestad, Jeen Broekstra, Navdha Agarwal, Pavel Mihailov, Prince Sachdeva, Raghav Rathi, Reeshabh Kumar Ranjan, Sakshi Saini, Shubhi Jain [TODO] 
```
